src/components/About/About.module.css

```diff
@@ -1,3 +1,172 @@
 .title {
   text-align: center;
 }
+
+@keyframes ldio {
+  0% {
+    opacity: 1;
+  }
+
+  100% {
+    opacity: 0;
+  }
+}
+
+.ldio div {
+  left: 94px;
+  top: 48px;
+  position: absolute;
+  animation: ldio linear 1s infinite;
+  background: #fe718d;
+  width: 12px;
+  height: 24px;
+  border-radius: 6px / 12px;
+  transform-origin: 6px 52px;
+  box-sizing: content-box;
+}
+
+.ldio div:nth-child(1) {
+  transform: rotate(0deg);
+  animation-delay: -.9166666666666666s;
+  background: #fe718d;
+}
+
+.ldio div:nth-child(2) {
+  transform: rotate(30deg);
+  animation-delay: -.8333333333333334s;
+  background: #fe718d;
+}
+
+.ldio div:nth-child(3) {
+  transform: rotate(60deg);
+  animation-delay: -.75s;
+  background: #fe718d;
+}
+  
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/todo_test/pullRequest?prid=5763907)